### DISCUSSION TITLE
Weird font display issue with LANG="zh_CN."

### DISCUSSION THREAD

**Discussion by 2403772980ygy** [kovidgoyal/kitty#6733] [Q&A] [answered]
When I set the locale to zh_CN.utf8, some of the default font changed. But this have an unforgivable impact on kitty, compared to others.

![maim](https://github.com/kovidgoyal/kitty/assets/40046418/6248ae67-bd10-47ce-abaf-782ecdb8db58)

The only font config for kitty is:

```
font_family	CascaydiaCove NF Regular
bold_font        CascaydiaCove NF Bold
italic_font	CascaydiaCove NF Italic
bold_italic_font CascaydiaCove NF Bold Italic
```

How should I solve this? I do have some chinese fonts installed on my system.

I've heard that changing fontconfig can affect the order of the fonts. But I have never tried it.

**Comment by kovidgoyal** [comment]
Use symbol_map in kitty.conf

**Reply by 2403772980ygy** [reply]
Well, I know that I can map unicode glyphs to other fonts with symbol_map, but without that, why were the normal glyphs interrupted?

I'll try to use symbol_map anyways, but I'm still quite curious.

**Comment by kovidgoyal** [comment]
because fontconfig is returning a font that has wide glyhs for a-z. Most
probably because the font name you have chosen doesnt exist or you have
a broken fontconfig configuration or who knows.

**Reply by 2403772980ygy** [reply]
Oh, maybe I'll have to manually configure fontconfig.

**Reply by kovidgoyal** [reply]
First dont use patched fonts, see https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font for how to use nerd fonts properly

**Reply by 2403772980ygy** [reply, marked as answer]
Solved by tweaking fontconfig settings.

TIL fontconfig, and I'll gradually switch away from patched font.
